frontend/src/container/GroupedNotificationItem.jsx

```diff
@@ -17,124 +17,150 @@ import {
+  // Issue : https://github.com/tracim/tracim/issues/5526
+  const handleReadGroupNotification = async (groupNotification) => {
+  //   // TODO - MP - 14-03-2022
+  //   // Create a props.dispatch(putGroupNotificationsAsRead(props.user.userId, groupNotification.group))
+  //   // Where it will read every notification in groupNotification.group
+  //   // related to https://github.com/tracim/tracim/issues/5526
+  //
+  //   const fetchPutNotificationAsRead = { status = 404 }
+  //   switch (fetchPutNotificationAsRead.status) {
+  //     case 204: {
+  //       props.dispatch(readGroupNotification(notificationId))
+  //       break
+  //     }
+  //     default:
+  //       props.dispatch(newFlashMessage(props.t('Error while marking the notification as read'), 'warning'))
+  //   }
   }
-}
 
-function handleClickGroupedNotification (props, e, notification) {
-  // INFO - GB - 2021-09-01 - A grouped notification is only updated on click if it's not grouped by content, in this case it just redirects to the content
-  if (notification.type.includes(GROUP_NOTIFICATION_CRITERIA.CONTENT)) {
-    notification.group.forEach(notification => props.onClickNotification(e, notification.id, {
-      url: PAGE.CONTENT(notification.content.type === TLM_SUB.COMMENT
-        ? notification.content.parentId
-        : notification.content.id)
-    }))
-  } else {
-    props.dispatch(updateNotification(notification.id, notification.group))
+  const handleClickGroupedNotification = (e, notification) => {
+    // NOTE - MP - 14-03-2022 - Since we redirect to the content if this is a group about a content
+    // we close the wall, then we ungroup the group
+    if (notificationDetails.url) {
+      props.onCloseNotificationWall()
+    }
+    setIsGrouped(false)
   }
-}
 
-const GroupedNotificationItem = props => {
-  const { notification, user } = props
-  const notificationDetails = getGroupedNotificationDetails(props, notification)
+  const { user } = props
+  const notificationDetails = getGroupedNotificationDetails(props.groupedNotifications)
   if (Object.keys(notificationDetails).length === 0) return null
 
-  const numberOfWorkspaces = uniqBy(notification.group.map(notification => notification.workspace), 'id').length
-  const numberOfAuthors = notification.author.length
-  const readStatus = notification.group.map(notification => notification.read)
+  const numberOfWorkspaces = uniqBy(props.groupedNotifications.group.map(notification => notification.workspace), 'id').length
+  const numberOfAuthors = props.groupedNotifications.author.length
+  const readStatus = props.groupedNotifications.group.map(notification => notification.read)
     .reduce((acc, current) => acc && current)
 
-  return (
+  const groupRender =
     <Link
       to={notificationDetails.url || '#'}
-      onClick={(e) => handleClickGroupedNotification(props, e, notification)}
+      onClick={(e) => handleClickGroupedNotification(props, e, props.groupedNotifications)}
```

Comment:
`handleClickGroupedNotification` only has 2 (unused) parameters in its signature… I think this part needs a bit of cleaning :-).